app/components/Feedback/index.tsx

```diff
@@ -27,57 +26,89 @@ type FeedbackProps = {
   pageid: string
   showForm?: boolean
   labels?: boolean
-  upHint?: string
-  downHint?: string
   options?: string[]
-  onSubmit?: (message: string, option?: string) => Promise<any>
+  onSubmit: (message: string, option?: string) => Promise<any>
 }
-const Feedback = ({
-  pageid,
-  showForm,
-  labels,
-  upHint,
-  downHint,
-  options,
-  onSubmit,
-}: FeedbackProps) => {
-  const [showFeedback, setShowFeedback] = useState(false)
+const Feedback = ({pageid, showForm, labels, options, onSubmit}: FeedbackProps) => {
+  const [showThanks, setShowThanks] = useState(false)
   const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [upHover, setUpHover] = useState(false)
+  const [downHover, setDownHover] = useState(false)
+  const [vote, setVote]: any = useState(undefined)
+
+  const thanksRef = useRef<HTMLDivElement | null>(null)
+
+  function sendVote(v: boolean) {
+    setVote(v)
+    // This is where you would post the vote to the db, except idk how to do that
+    // If 'v' is true, then user upvoted
+    // if 'v' is false, then user downvoted
+  }
 
   useEffect(() => {
-    const timeout = setInterval(() => setShowFeedback(false), 6000)
-    return () => clearInterval(timeout)
-  }, [showFeedback])
+    if (showThanks && thanksRef.current) {
+      thanksRef.current.style.opacity = '1'
+      setTimeout(() => {
+        if (thanksRef.current) thanksRef.current.style.opacity = '0'
+      }, 6000)
+    }
+  }, [showThanks])
 
   return (
-    <div className="feedback relative">
-      <CompositeButton className="flex-container">
-        <Action
-          pageid={pageid}
-          showText={!!labels}
-          actionType={ActionType.HELPFUL}
-          hint={upHint}
-          onClick={() => setShowFeedback(true)}
-        />
-        <Action
-          pageid={pageid}
-          showText={!!labels}
-          hint={downHint}
-          actionType={ActionType.UNHELPFUL}
+    <div className="relative flex items-center">
+      <div className="flex rounded-md border-[1px] border-[#DFE3E9] w-fit p-[6px]">
+        <button
+          disabled={vote !== undefined}
+          onClick={() => {
+            sendVote(true)
+            setShowThanks(true)
+          }}
+          className={
+            'p-1 rounded-[4px] relative ' +
+            (vote === true ? 'bg-[#EDFAF9]' : vote === undefined ? 'hover:bg-[#F9FAFC]' : '')
+          }
+          onMouseEnter={() => setUpHover(true)}
```

Comment:
why not just do this in css? The `:hover` pseudo class is for that - no point in using js where css can do the job better

Reply:
You are totally right, I've just fixed it